app/lib/person_disposable_income_subtotals.rb

```diff
@@ -68,11 +68,11 @@ def rent_or_mortgage_bank
   end
 
   def rent_or_mortgage_cash
-    @disposable.rent_or_mortgage_cash
+    @outgoings.housing_costs.gross_housing_costs_cash
   end
 
   def rent_or_mortgage_all_sources
-    rent_or_mortgage_bank + rent_or_mortgage_cash + @regular.rent_or_mortgage_regular
+    rent_or_mortgage_bank + rent_or_mortgage_cash + @outgoings.housing_costs.gross_housing_costs_regular
```

Comment:
@starswan Should we create a method `rent_or_mortgage_regular` ?

Reply:
not sure it would add much - we don't expose this field (so it would be private) and it make sense to do it everywhere. I think this is showing that this object should not have this much knowledge ?